src/main/java/li/cil/tis3d/client/gui/GuiModuleTerminal.java

```diff
@@ -0,0 +1,99 @@
+package li.cil.tis3d.client.gui;
+
+import li.cil.tis3d.api.util.RenderUtil;
+import li.cil.tis3d.common.Settings;
+import li.cil.tis3d.common.module.ModuleTerminal;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiScreen;
+//import net.minecraft.client.renderer.GlStateManager;
+//import net.minecraft.util.EnumHand;
+import net.minecraftforge.client.event.RenderGameOverlayEvent;
+import net.minecraftforge.common.MinecraftForge;
+import cpw.mods.fml.common.eventhandler.SubscribeEvent;
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import org.lwjgl.input.Keyboard;
+import org.lwjgl.opengl.GL11;
+
+import java.io.IOException;
+
+/**
+ * Invisible GUI for the terminal module, purely used to grab keyboard input.
+ */
+@SideOnly(Side.CLIENT)
+public class GuiModuleTerminal extends GuiScreen {
+    private final ModuleTerminal module;
+
+    public GuiModuleTerminal(final ModuleTerminal module) {
+        this.module = module;
+    }
+
+    public boolean isFor(final ModuleTerminal that) {
+        return that == module;
+    }
+
+    @Override
+    public void drawScreen(final int mouseX, final int mouseY, final float partialTicks) {
+        //GlStateManager.disableTexture2D();
```

Comment:
Did you deliberately not port those to their corresponding GL calls (because it caused bugs in 1.7.10 or something)?